dev-docs/modules/userId.md

```diff
@@ -815,11 +815,26 @@ This module stores an unique user id in the first party domain and makes it acce
 There is no special registration or configuration for PubCommon ID. Each publisher's privacy policy should take
 PubCommon ID into account.
 
+PubCommon ID now has built-in support for Shared ID.  No configuration change is required, nor does the sharedIdSystem module need to be included in the build.  When a new PubCommon ID is generated, the module will make a call to sharedid.org to obtain a Shared ID, and cache it under the same storage configuration as PubCommon ID.  It is possible to include both pubCommondIdSystem and the main sharedIdSystem modules. 
```

Comment:
Per the note above on line 815/816 about privacy policy...do publishers need to do list shared ID now that it's being automatically included? What if a publisher doesn't want to use sharedId, or can't for a legal/privacy perspective?

I think this section should also be highlighted more than just a paragraph, so that publishers don't skip over it when setting up pubcommon.

Reply:
I think this is sufficient @smenzer, the text notes that Pubcommon now includes built in support for SharedId.   https://sharedid.org/optout is where users can elect to opt-out.  If we wanted to include that link to the text I'd be fine with that.  I plan to make an edit to the Sharedid module docs today doing the same.